modules/ROOT/pages/create-reference-map.adoc

```diff
@@ -1,10 +1,10 @@
 = Creating Reference Maps for Message Types
 
-When you create a new Source at Host message type or a Receive from Partners message type formatted with JSON or XML, you must associate the message type with a DataWeave reference map. This is optional for Receive from Partners message types formatted EDFIACT or X12. Reference maps can include custom message attributes that get extracted from the message payloads at runtime. The extracted attributes are visible in the payload and are available for searching and tracking.
+When you create a new Source at Host or Receive from Partners message type with a CSV, JSON, or XML format, you must associate the message type with a DataWeave reference map. This is optional for Receive from Partners message types with an EDFIACT or X12 format. Reference maps can include custom message attributes that get extracted from the message payloads at runtime. The extracted attributes are visible in the payload and are available for searching and tracking.
```

Comment:
Reword "This is" per [SG](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22).

Reply:
```suggestion
When you create a new Source at Host or Receive from Partners message type with a CSV, JSON, or XML format, you must associate the message type with a DataWeave reference map. This is optional for Receive from Partners message types with an EDFIACT or X12 format. Reference maps can include custom message attributes that are extracted from the message payloads at runtime. The extracted attributes are visible in the payload and are available for searching and tracking.
```

Reply:
I can't think of a good way to reword this.